telco-hub/configuration/reference-crs-kube-compare/ReferenceVersionCheck.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: config.openshift.io/v1
+kind: ClusterVersion
+metadata:
+  name: version
+status:
+  desired:
+    version: {{ template "versionMatch" (list .status.desired.version "4.18") }}
```

Comment:
Current "main" branch should be 4.19

Reply:
The reference-crs still point to channel versions related to 4.18. I guess both the reference-crs and the templates should be updated at once.

Reply:
There are a couple of changes for 4.19 which are pending based on testing. Until we merge those we are trying to keep main at 4.19 to make things easier.